front/se_project/src/Pages/viewExpertProfile.jsx

```diff
@@ -0,0 +1,54 @@
+import {Component} from 'react';
+import Button from '../components/button';
+import Input_text from '../components/input_form';
+import Breadcrumbb from '../components/breadcrumb';
+import Navbar from '../components/navBar/navBar';
+import Footer from '../components/Footer/footer';
+import HorizontalLine from '../components/horizontalLine/horizontalLine';
+import ExpertResume from '../components/expertResume';
+import ShowArtworkAdCost from '../components/ArtworkAdInfo/showArtworkAdCost';
+import x from "../assets/ig.jpg";
+
+const ViewExpertProfile = (props) => {
+
+    return (
+        <div className='d-flex flex-column min-vh-100'>
+            <Navbar/>
+            <Breadcrumbb is3OrNot='true' first='صفحه اصلی' second='آگهی اثر' third='نمایش حساب کاربری کارشناس'/>
+            <HorizontalLine value='اطلاعات فردی'/>
+            <form style={{backgroundColor:'#F9FFFF'}}>
+                <div className='container w-75 rounded'>
+                    <div className='row justify-content-center'>
+                        <div class="row position-relative" dir="LTR"  >
+                            <div class="col-md-5 p-md-4">
+                                    <img class="img-fluid" src={x}/*get the first img from back */ alt="profile image"  style={{blockSize: "9vw", maxWidth:'100%',maxHeight:'100%',marginLeft:'5vw'}}/>
```

Comment:
I think in prototype we don't have profile image for expert! I think you should delete this tag.

Reply:
I can delete the image but in user information there is no info to show.
we can add this.